addons/sms/models/sms_template.py

```diff
@@ -157,3 +157,7 @@ def _get_translated_bodies(self, res_ids):
     def _render_template(self, template_txt, model, res_ids):
         """ Render the jinja template """
         return self.env['mail.template']._render_template(template_txt, model, res_ids)
+
+    def unlink(self):
+        self.mapped('sidebar_action_id').unlink()
+        return super().unlink()
```

Comment:
return super(SMSTemplate, self).unlink()

Reply:
Is there a reason why `SMSTemplate, self` would be needed? It should work fine with Python 3.

Reply:
@nim-odoo
1) `return super(Class, self).unlink()` is used all over the place in Odoo so keeping it is more consistent
2) Retro-compatible with P2

Reply:
@Yenthe666 

1. So is `super()`: 253 occurrences in Odoo.
2. Lol.

Reply:
We don't write code looking like JS dummies. I did not even know super().method() worked and I think it is a terrible way of writing code.

Reply:
@tde-banana-odoo tu te coucheras moins bête.

Reply:
@nim-odoo  Ca c'est pas sûr par contre. Le naufrage de la vieillesse !